### ISSUE TITLE
 p = w0 + dataMatrix[x]*w + interaction             loss = sigmoid(classLabels[x] * p[0, 0]) - 1

### ISSUE THREAD

**Issue by aMIT-feat-aMEI** [GreenArrow2017/MachineLearning#1]
Factorization machine 里的一行代码，p[0,0] 表示什么意思

**Comment by caojiangxia** [comment]
表示的是预测结果啊。在0-1之间。
